package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fury-adapter-swagger",
-  "version": "0.18.4",
+  "version": "0.18.5",
```

Comment:
This should be 0.19.0 because the changes update the peer dependency which is not semantically permitted.

Reply:
Updated